fineract-provider/src/main/java/org/apache/fineract/adhocquery/domain/AdHoc.java

```diff
@@ -58,21 +65,6 @@ public final class AdHoc extends AbstractAuditableCustom {
     @Column(name = "is_active", nullable = false)
     private boolean isActive = false;
```

Comment:
2 things here:
- I think this variable should be named just "active"; because it is a base type (boolean) Lombok will generate a "getter function" that is called "isActive()"... which is better style... the prefix "is" in the variable is kind of redundant, so let's try to get rid of it
- also: as this is a base type "boolean" the default value is anyway "false" so let's remove that unnecessary assignment

The rest looks good and after these small changes we should be ready to merge.

Reply:
corrected